### ISSUE TITLE
A11y_code.visualstudio.com_Docs_Left Right Navigation_ScreenReader: Screen reader is not announcing that link is selected or not selected.

### ISSUE THREAD

**Issue by RohitJadhav-TCS** [microsoft/vscode-docs#2786]
“[Check out Accessibility Insights](https://nam06.safelinks.protection.outlook.com/?url=https%3A%2F%2Faccessibilityinsights.io%2F&data=02%7C01%7Cv-manai%40microsoft.com%7Cb67b2c4b646d4f9561a208d6f4b5c39b%7C72f988bf86f141af91ab2d7cd011db47%7C1%7C0%7C636965458847260936&sdata=T26HQfSGOlnuRQdX%2ByXk%2B2bxqgwFvCIVfuboZUWidYY%3D&reserved=0)! - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**Environment Details:**
URL: [https://code.visualstudio.com](https://code.visualstudio.com)
Windows Version: Windows10
Browser Version:
Microsoft Edge 44.18362.1.0
Chrome Version 75.0.3770.100 (Official Build) (64-bit)
Screen Reader Versions : NVDA2019.1.1
MAS Violated: MAS 1.3.1

**Repro Steps:**
Navigate to the URL: [https://code.visualstudio.com/docs](https://code.visualstudio.com/docs)
follow below steps:
1.Launch the application URL: https://code.visualstudio.com
2.Navigate to "Docs" link.
3.Left and right navigation will appear.
4.Navigate to any link and activate it.
5.Start the screen reader.
6.Navigate to same link again.

**Actual:**
Screen reader is not announcing that link is selected or not. 
It is announcing same information for Selected and non selected link.

This issue is observed with Edge+Narrator and Chrome+NVDA

**Expected:**
Screen reader should announce current state that it is currently selected.

**Recommendations:**
Refer below link which is repository of bug fixes code snippets:
[https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx](https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx)

**User Impact:**
Screen reader user will get confused while navigating through link.
On left panel if we have selected on particular link the it will get bold and content of that link will get displayed in main section but at that time if user navigate again through left navigation section then user is unable to differentiate between selected and non selected link. 

**MAS Reference**:
MAS 1.3.1 -
[https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4})

**Attachment for Reference:**
[Code.visualstudio.com_Docs_Left Right Navigation_ScreenReader is not announcing that link is selected or not selected.pptx](https://github.com/microsoft/vscode-docs/files/3314482/Code.visualstudio.com_Docs_Left.Right.Navigation_ScreenReader.is.not.announcing.that.link.is.selected.or.not.selected.pptx)
